scripts/server

```diff
@@ -1,32 +1,23 @@
-#! /usr/bin/env python
+#!/bin/bash
 
-import os
-import logging
-from BaseHTTPServer import HTTPServer
-from SimpleHTTPServer import SimpleHTTPRequestHandler
+set -e
 
-PORT = 54633
-PATH_TO_PROJECT = 'src/GeositeFramework/'
+if [[ -n "${GEOSITE_DEBUG}" ]]; then
+    set -x
+fi
 
-logging.basicConfig(level=logging.INFO)
+function usage() {
+    echo -n "Usage: $(basename "$0")
+    Starts the python development server.
```

Comment:
Consider specifying that it's starting a Docker container Python development server.